messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/ServerAdd.java

```diff
@@ -329,18 +329,12 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
             // Inject a reference to the Elytron security domain if one has been defined.
             Optional<Supplier<SecurityDomain>> elytronSecurityDomain = Optional.empty();
             // legacy security
-            Optional<Supplier<SecurityDomainContext>> securityDomainContext = Optional.empty();
             final ModelNode elytronSecurityDomainModel = ELYTRON_DOMAIN.resolveModelAttribute(context, model);
             if (elytronSecurityDomainModel.isDefined()) {
                 ServiceName elytronDomainCapability = context.getCapabilityServiceName(ELYTRON_DOMAIN_CAPABILITY, elytronSecurityDomainModel.asString(), SecurityDomain.class);
                 elytronSecurityDomain = Optional.of(serviceBuilder.requires(elytronDomainCapability));
             } else {
-                // Add legacy security
-                String domain = SECURITY_DOMAIN.resolveModelAttribute(context, model).asString();
-                securityDomainContext = Optional.of(serviceBuilder.requires(SECURITY_DOMAIN_SERVICE.append(domain)));
-                // WFLY-6652 / WFLY-10292 this dependency ensures that Artemis will be able to destroy any queues created on behalf of a
-                // pooled-connection-factory client during server stop
-                serviceBuilder.requires(SECURITY_BOOTSTRAP_SERVICE);
+                throw ROOT_LOGGER.legacySecurityUnsupported();
```

Comment:
I think this is 'wrong' as it would require an elytron domain to be always configured. I think we should incorporate https://github.com/wildfly/wildfly/pull/14790 in there.

Reply:
+1 it is wrong, but no more wrong than the current codebase i.e. in WildFly 26 Beta if we configure to reach this section the service would not be available so I wanted to throw an error rather than quietly ignore and accidentally leave unsecured.

But yes I think we need to get your PR in and then this one can be rebased and likely eliminate this error.